scripts/release-s3.sh

```diff
@@ -79,7 +79,7 @@ if [[ "$CHANNEL" == "nightly" ]]; then
     "https://packages.timber.io/vector/nightly/latest/vector-nightly-x86_64-unknown-linux-gnu.tar.gz" \
     "$td_nightly/vector-nightly-x86_64-unknown-linux-gnu.tar.gz"
 
-elif [[ "$CHANNEL" == "latest" ]]; then
+elif [[ "$CHANNEL" == "release" ]]; then
```

Comment:
Matches new release channel name (used to be `latest`).

Reply:
🤔 when did that channel name change?

Reply:
I think it was part of https://github.com/vectordotdev/vector/pull/17024 since it unified the `channel` and `mode` parameters.

Reply:
I was afraid of that...